java/README.md

```diff
@@ -59,6 +59,19 @@ For any other library, such as OkHttp, you will need to include the correspondin
 from the [instrumentation project](https://github.com/open-telemetry/opentelemetry-java-instrumentation) and
 modify your code to initialize it in your function.
 
+### Configuring Context Propagators
+
+#### If you emit your traces to AWS X-Ray (instead of a third-party service) and have enabled X-Ray Active Tracing
+Please use the following environment variable for your lambda:
+`OTEL_PROPAGATORS=tracecontext,baggage,xray-lambda`
+
+#### If you emit your traces to another system besides AWS X-Ray (Default Setting)
+Please use the following environment variable for your lambda:
```

Comment:
```suggestion
The following propagators are configured by default, but you can use this environment variable to customize it:
```

Reply:
I like that better. Ill submit another revision

Reply:
(You can just commit my suggestion directly in the UI if you want.)

Reply:
ah nice, just did it. thanks!